Core/DISET/private/Transports/SSL/SocketInfo.py

```diff
@@ -65,8 +65,8 @@ def gatherPeerCredentials( self ):
         identitySubject = peerChain.getIssuerCert()['Value'].getSubjectNameObject()[ 'Value' ]
```

Comment:
Regarding this file: is it the only one that needs to be changed for moving to M2Crypto?

Reply:
Unfortunately no. This is only to fix a crash.